server/app/controllers/admin/AdminImportController.java

```diff
@@ -74,7 +79,7 @@ public Result hxImportProgram(Http.Request request) {
         formFactory
             .form(AdminProgramImportForm.class)
             .bindFromRequest(request, AdminProgramImportForm.FIELD_NAMES.toArray(new String[0]));
-    String jsonString = form.get().getProgramJson();
+    String jsonString = form.get().getProgramJSON();
```

Comment:
**nit**: I feel bad about nitting this but the styleguide says it should be `getProgramJson()`:
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
Thank you for the nit! I should have checked the style guide, I wasn't sure.

Reply:
Updated.